File: src/main/java/com/hjj/apiserver/service/SavingService.java
package com.hjj.apiserver.service;

import com.hjj.apiserver.dto.DepositDto;
import com.hjj.apiserver.dto.SavingDto;
import com.hjj.apiserver.repositroy.SavingRepository;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
@RequiredArgsConstructor
public class SavingService {

    private final SavingRepository savingRepository;


    public List<SavingDto.ResponseSavingFindAll> findSavingList(){

        return savingRepository.findSavingAll();
    }
}

File: src/main/kotlin/com/hjj/apiserver/service/SavingService.kt
package com.hjj.apiserver.service

import com.hjj.apiserver.dto.saving.response.SavingFindAllResponse
import com.hjj.apiserver.repository.saving.SavingRepository
import org.springframework.stereotype.Service

@Service
class SavingService(
    private val savingRepository: SavingRepository,
) {

    fun findSavings():List<SavingFindAllResponse>{
        return savingRepository.findSavingAll()
    }


}